src/test/java/com/browserstack/SingleTest.java

```diff
@@ -5,17 +5,27 @@
 
 import org.testng.Assert;
 import org.testng.annotations.Test;
+import org.openqa.selenium.JavascriptExecutor;
 
 public class SingleTest extends BrowserStackTestNGTest {
 
     @Test
     public void test() throws Exception {
         driver.get("https://www.google.com/ncr");
+        try {
+            WebElement ele = driver.findElement(By.className("IKl7Q"));
+            if(ele.isDisplayed()) {
+                ((JavascriptExecutor)driver).executeScript("arguments[0].scrollTo(0, arguments[0].scrollHeight)",ele);
+                driver.findElement(By.xpath("//*[@id='L2AGLb']/div")).click();
```

Comment:
these classnames and ids might change in the future with their deploys since they are generated for creating unique values from original classnames.

Will require some other selectors.